components/TodayPerHour.js

```diff
@@ -34,4 +22,25 @@ const styles = StyleSheet.create({
     alignItems: 'center',
   },
 });
+
+const TodayPerHour = (props) => {
+  const { weather } = props;
```

Comment:
Use destructuring in function parameters.

Reply:
Always one line less